assets/scss/templates/_guidelines.scss

```diff
@@ -212,7 +145,7 @@ body {
 .dpsg-main-wrap {
   display: flex;
   flex-direction: column;
-  font-family: $dp-font-family-proximanova;
+  font-family: Arial, sans-serif;
```

Comment:
How about adding these font types as a variable too? maybe we reuse them somewhere else.

Reply:
_$ dp-font-family: Arial, sans-serif! default;
$ dp-font-family-proximanova: 'proxima-nova', Helvetica, Arial, sans-serif! default;
$ dp-font-family-handofsean: 'hand-of-sean';_

**If we have them I forgot to put it**